networks/ligand_networks_for_developers.ipynb

```diff
@@ -0,0 +1,895 @@
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "mappings[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "82f387dd",
+   "metadata": {},
+   "source": [
+    "Bad inputs (meaning unable to form an edge) fail silently here:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "17cba630",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[]"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "list(mapper.suggest_mappings(methanol, propane))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d47b37b5",
+   "metadata": {},
+   "source": [
+    "### Returning multiple mappings\n",
+    "\n",
+    "OpenFE's `LigandAtomMapper` is designed to return multiple candidate mappings, which may then be scored by a scorer to select the best one. Let's modify the previous code so that instead of setting `remove_middle` to either `True` or `False` in the `__init__`, we have it return both options for that parameter."
```

Comment:
So I follow this well, but I do wonder how big a logical step it is to understand. Ideally we should have some kind of figure explaining how the API looks. Maybe an issue to be resolved in a future iteration?